requirements.txt

```diff
@@ -1,2 +1,2 @@
-rich
-htcondor
+rich>=6.1.1
+htcondor>=8.9.8
```

Comment:
This is an development release. Unless there's needed functionality in the 8.9.x series, I would prefer requiring `>=8.8.6`.

Reply:
Fixed in a later commit now.